File: stepik-java-api/src/main/java/org/stepik/api/actions/StepikReviewSessionsAction.java
package org.stepik.api.actions;

import org.jetbrains.annotations.NotNull;
import org.stepik.api.client.StepikApiClient;
import org.stepik.api.objects.ObjectsContainer;
import org.stepik.api.queries.reviews.StepikReviewSessionsPostQuery;

/**
 * @author meanmail
 */
public class StepikReviewSessionsAction extends StepikAbstractAction {
    public StepikReviewSessionsAction(@NotNull StepikApiClient stepikApiClient) {
        super(stepikApiClient);
    }

    @NotNull
    public <R extends ObjectsContainer> StepikReviewSessionsPostQuery post() {
        return new StepikReviewSessionsPostQuery(this);
    }
}

File: stepik-java-api/src/main/kotlin/org/stepik/api/actions/StepikReviewSessionsAction.kt
package org.stepik.api.actions

import org.stepik.api.client.StepikApiClient
import org.stepik.api.objects.ObjectsContainer
import org.stepik.api.queries.reviews.StepikReviewSessionsPostQuery

class StepikReviewSessionsAction(stepikApiClient: StepikApiClient) : StepikAbstractAction(stepikApiClient) {
    
    fun <R : ObjectsContainer<*>> post(): StepikReviewSessionsPostQuery {
        return StepikReviewSessionsPostQuery(this)
    }
}
